{"name": "Проверка невозможности изменения данных авторизованным пользователем с чужой почтой", "status": "passed", "description": "Проверяем, что авторизованный пользователь не может поменять данные, указав чужую почту", "start": 1717323798214, "stop": 1717323798523, "uuid": "3324d403-2cfc-4704-84ea-473039f1143b", "historyId": "db2bda4e1f83ac4c53b810c918fcf0b6", "testCaseId": "db2bda4e1f83ac4c53b810c918fcf0b6", "fullName": "tests.test_change_data_user.TestChangeDataUser#test_change_data_existing_email", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_change_data_user"}, {"name": "subSuite", "value": "TestChangeDataUser"}, {"name": "host", "value": "LAPTOP-SSVUBLG9"}, {"name": "thread", "value": "12652-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_change_data_user"}]}